CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+### New features
+
+- `@inrupt/solid-client/access/universal` and `@inrupt/solid-client/access/for`:
+  A set of APIs to manage Access which are agnostic to the Access Control mechanism
```

Comment:
Also, maybe we should mention here too that this is a preview release?

```suggestion
- `@inrupt/solid-client/access/universal`: an early preview of
  a set of APIs to manage Access which are agnostic to the Access Control mechanism
```

Possibly even noting the fact that there are some bugs that are yet to be fixed, but that they only occur in a limited number of edge cases involving existing Access Policies.